src/draw_buffer.rs

```diff
@@ -149,127 +153,130 @@ impl DrawBuffer {
             &ColorAttachment::Texture(id) => Some(id),
         }
     }
-}
-
-// NOTE: The initially associated GLContext MUST be the current gl context
-// when drop is called. I know this is an important constraint.
-// Right now there are no problems, if not, consider using a pointer to a
-// parent with Rc<GLContext> and call make_current()
-impl Drop for DrawBuffer {
-    fn drop(&mut self) {
-        gl::delete_framebuffers(&[self.framebuffer]);
 
-        // NOTE: Color renderbuffer is destroyed on drop of
-        //   ColorAttachment
-        gl::delete_renderbuffers(&[self.stencil_renderbuffer, self.depth_renderbuffer]);
+    fn gl(&self) -> &gl::Gl {
+        // See the comment on top the GLContext field to see why our usage of
```

Comment:
perhaps, there is no longer a point in this comment? There is no `unsafe` here, so safety is assumed by default